Tests.sql

```diff
@@ -645,7 +650,12 @@ ROLLBACK TRANSACTION
 GO
 
 --Mislukking
---[S00016][50001] Een project kan niet meer aangepast worden nadat deze is afgelopen.
+/*
+CATCH RESULTATEN:
+ERROR NUMMER:		50001
+ERROR SEVERITY:	16
+ERROR MESSAGE:	Een project kan niet meer aangepast worden nadat deze is afgelopen.
+*/
 BEGIN TRANSACTION
 BEGIN TRY
 IF (select IDENT_CURRENT('medewerker_op_project')) IS NOT NULL
```

Comment:
Is dit nog nodig aangezien we me IDENT_CURRENT gebruiken

Reply:
dit is lastig te testen nu zo laat, want er zijn verschillende procedures en rechten die ik aan moet zetten om dit te testen =/

Reply:
bij de tests waar de date met een declare vervangen kon worden heb ik dit gedaan, maar sommige heb ik niet aangepast omdat ze te veel procedures en tests nodig heeft.

Reply:
👍